### ISSUE TITLE
Error starting blocky v0.21

### ISSUE THREAD

**Issue by c-st** [tteck/Proxmox#1361]
### A clear and concise description of the issue.

blocky in its current version (v0.21) does not start inside the container. It seems to be due to requiring glibc-2.32 which is apparently [not available on Debian bullseye](https://tracker.debian.org/pkg/glibc).

Are there suggestions on how to deal with this? The only obvious way I see is changing the underlying OS. Is it also possible to specify the value for `RELEASE` manually from the script `blocky.sh`?

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

```
root@blocky:/opt/blocky# systemctl status blocky
* blocky.service - Blocky
     Loaded: loaded (/etc/systemd/system/blocky.service; enabled; vendor preset: enabled)
     Active: failed (Result: exit-code) since Sun 2023-04-23 15:33:39 CEST; 28min ago
   Main PID: 1952 (code=exited, status=1/FAILURE)
        CPU: 1ms

Apr 23 15:33:39 blocky systemd[1]: Started Blocky.
Apr 23 15:33:39 blocky blocky[1952]: /opt/blocky/./blocky: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /opt/blo>
Apr 23 15:33:39 blocky blocky[1952]: /opt/blocky/./blocky: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /opt/blo>
Apr 23 15:33:39 blocky systemd[1]: blocky.service: Main process exited, code=exited, status=1/FAILURE
Apr 23 15:33:39 blocky systemd[1]: blocky.service: Failed with result 'exit-code'.
```

```
root@blocky:/opt/blocky# ldd --version
ldd (Debian GLIBC 2.31-13+deb11u5) 2.31
```

### Please provide detailed steps to reproduce the issue.

Setup new blocky container:
`bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/blocky.sh)"`

Verify blocky status from inside container:
`systemctl status blocky`

**Comment by tteck** [comment]
https://github.com/0xERR0R/blocky/issues/983

**Comment by tteck** [comment]
Pinned to version `0.20` until issue gets resolved in `0.22`

**Comment by c-st** [comment]
Thanks for the pointer. Looks like they're going to fix it with their next 0.22 release then. 

Until then I'll just install the previous version manually:
```
wget -q https://github.com/0xERR0R/blocky/releases/download/v0.20/blocky_0.20_Linux_x86_64.tar.gz
tar -xf blocky_0.20_Linux_x86_64.tar.gz -C /opt/blocky
rm blocky_0.20_Linux_x86_64.tar.gz
```
